packages/ui/components/avatar/Avatar.tsx

```diff
@@ -50,7 +50,10 @@ export function Avatar(props: AvatarProps) {
           alt={alt}
           className={classNames("aspect-square rounded-full", sizesPropsBySize[size])}
         />
-        <AvatarPrimitive.Fallback delayMs={600} asChild={props.asChild} className="flex items-center">
+        <AvatarPrimitive.Fallback
+          delayMs={600}
+          asChild={props.asChild}
+          className="flex h-full items-center justify-center">
```

Comment:
Only thing changed is the added class `h-full` to better support an icon fallback, like the "+" in the org creation flow on the about part